Feat/kennisbank video audiodescriptie (#2340)
nl-design-system/documentatie: docs/richtlijnen/content/video/audiodescriptie.md

---
title: Audiodescriptie · Content · Richtlijnen
hide_title: true
hide_table_of_contents: false
sidebar_label: Audiodescriptie
pagination_label: Audiodescriptie
description: Richtlijnen voor Audiodescriptie binnen NL Design System
slug: /richtlijnen/content/video/audiodescriptie
keywords:
  - video
  - youtube
  - vimeo
  - audiodescriptie
  - audiodescription
  - audio
  - beschrijving bij video
  - video beschrijving
  - videobeschrijving
---

<!-- @license CC0-1.0 -->

import { Canvas } from "@site/src/components/Canvas/Canvas";
import { Guideline } from "@site/src/components/Guideline";
import FooterInfo from "@nl-design-system-unstable/documentation/richtlijnen/\_footer_info.md";

## Audiodescriptie bij video

> Een audiodescriptie bij een video is een hoorbare gesproken beschrijving van belangrijke visuele informatie die niet voorkomt in de audio van de video. Het bevat:

Reviewer: Wat vind je van:
```suggestion
Audiodescriptie bij een video is een hoorbare, gesproken beschrijving van belangrijke informatie die je ziet in een video, maar niet hoort. Voorbeelden hiervan zijn:
```

De eerste zin lijkt me misschien iets leesbaarder zo? En "Het bevat" lijkt me te definitief, dus ik dacht aan iets met "voorbeelden hiervan" of "bijvoorbeeld"? Open suggestie.

- Informatie over tekst in beeld (zoals de namen en functies van sprekers, slogans en links).
- Belangrijke visuele informatie in video, zoals de acties en kenmerken van personages, gebeurtenissen en scènewisselingen.

## Voordelen

> Het aanbieden van een audiodescriptie bij een video is vooral belangrijk voor blinde en slechtziende bezoekers - zij kunnen de video niet of niet goed zien en missen belangrijke informatie als er geen audiodescriptie aanwezig is.

Reviewer: Is het "een audiodescriptie" of is het "audiodescriptie" zonder dat "een" nodig is? Algemene vraag voor het hele artikel

Author: hmm, dan ga ik voor zonder 'een'!

Daarnaast hebben ook andere doelgroepen, zoals mensen met een cognitieve beperking, baat bij audiodescriptie. Bijvoorbeeld omdat zij audio makkelijker kunnen verwerken dan visuele informatie, of omdat zij moeite hebben met het interpreteren van bijvoorbeeld gezichtsuitdrukkingen.

## Is audiodescriptie altijd nodig?

> Je voegt audiodescriptie toe wanneer iemand zonder de informatie die wel in beeld is, maar niet in het audiospoor voorkomt, de video niet volledig kan begrijpen. Dat is niet bij iedere video het geval, bijvoorbeeld:

Reviewer 2: Deze zin vind ik lastig te volgen.

Author: ingekort!

- Wanneer de informatie in de audio al overeenkomt met de informatie in beeld.

> - Wanneer de informatie in beeld gedurende de video niet verandert. Bijvoorbeeld: rr is maar één spreker en de spreker kijkt en spreekt gedurende de video rechtstreeks in de camera (een zogenaamde 'talking head-video').

Author: dank!

> Als er hiernaast andere informatie voorkomt in de video, zoals tekst die in beeld komt, is een audiodescriptie natuurlijk alsnog wél nodig.

Reviewer: ```suggestion
Als er daarnaast andere informatie voorkomt in de video, zoals tekst die in beeld komt, is een audiodescriptie natuurlijk alsnog wél nodig.
```

### Video als alternatief voor tekst

> In de [WCAG-richtlijnen](/wcag) is een uitzondering opgenomen voor video als alternatief voor tekst, als er duidelijk wordt aangegeven dát de video een alternatief is voor tekst. (**Let op**: dit geldt dus niet andersom - een tekstalternatief voor video, zoals een transcript, is nooit een toegankelijk alternatief voor een video). De informatie in tekst en in de video moeten dan wel overeenkomen - je mag geen nieuwe of afwijkende informatie toevoegen.

Reviewer 3: Deze zin is wel weer lang. Mijn suggestie zou óók hier weer zijn: opsplitsen. Minder komma's is meer beter.

## Audiodescriptie als onderdeel van de video of als los bestand

Veel mensen hebben baat bij audiodescriptie, maar voor een deel van je bezoekers is audiodescriptie juist erg overprikkelend. Daarom is het aanbieden van audiodescriptie als los bestand in de videospeler een gebruiksvriendelijke keuze. Zo kan je bezoeker zelf kiezen om er gebruik van te maken, net als bij [gesloten ondertiteling](/richtlijnen/content/video/ondertiteling#gesloten-en-open-ondertiteling). Of je de audiodescriptie als los audiobestand kunt toevoegen, hangt af van welke videospeler je organisatie gebruikt.

Daarnaast kun je de audiodescriptie integreren in de video. Dit houdt dan wel in dat álle bezoekers de audiodescriptie horen. Je kunt er ook voor kiezen om dezelfde video als een versie mét en een versie zónder audiodescriptie aan te bieden.

### Tekst-naar-spraak of audio

Als je de audiodescriptie als los bestand aanbiedt, heb je twee opties: de audiodescriptie aanbieden als tekst-naar-spraak of als audio.

> Als de videospeler tekst-naar-spraak bestanden ondersteunt, is dit meestal met een [WebVTT bestand](https://en.wikipedia.org/wiki/WebVTT). Dit is een tekstbestand met tijdstempels, waardoor de audiodescriptie op het juiste moment wordt voorgelezen. Dit is een relatief makkelijke en snelle manier om audiodescriptie te maken, maar de kwaliteit van de voorleesstem is afhankelijk van de videospeler.

Reviewer: Gebaseerd op de zoekresultaten in de code op "-bestand":
```suggestion
Als de videospeler tekst-naar-spraak bestanden ondersteunt, is dit meestal met een [WebVTT-bestand](https://en.wikipedia.org/wiki/WebVTT). Dit is een tekstbestand met tijdstempels, waardoor de audiodescriptie op het juiste moment wordt voorgelezen. Dit is een relatief makkelijke en snelle manier om audiodescriptie te maken, maar de kwaliteit van de voorleesstem is afhankelijk van de videospeler.
```

> Je kunt de audiodescriptie ook toevoegen als audiobestand. Je neemt dan een extra audiobestand op met een eigen voorlezer. Deze methode vraagt meer tijd en is mogelijk ook duurder, maar je hebt de kwaliteit dan wel volledig zelf in de hand. Je levert het audiobestand aan in een formaat dat ondersteund wordt door je videospeler, zoals een [MP3](https://nl.wikipedia.org/wiki/MP3) of [WAV](https://nl.wikipedia.org/wiki/WAV) bestand.

Author: @eslook ik twijfelde hier al over, dan moet ik het eigenlijk overal met streepje doen, ook bij WBVTT

- [Beschrijvingen van belangrijke afbeeldingen](/richtlijnen/content/afbeeldingen) die in beeld komen, zoals het logo van de organisatie.
- Beschrijvingen van belangrijke gebeurtenissen die in beeld komen, bijvoorbeeld informatie over wat mensen aan het doen zijn.
- Gezichtsuitdrukkingen en lichaamstaal, als dat relevant is.

Deze informatie kun je vervolgens ook gebruiken bij het samenstellen van een [transcript voor de video.](/richtlijnen/content/video/transcript)

## Help, ik heb geen ruimte voor audiodescriptie!

Het kan lastig zijn om audiodescriptie toe te voegen aan een video, zeker als het om een bestaande video gaat. Gelukkig zijn er een aantal concrete stappen die je kunt nemen om ruimte te maken voor audiodescriptie.

### Audiodescriptie voor een nieuwe video

Ga of laat je een nieuwe video maken? Door vooraf rekening te houden met audiodescriptie maak je de audiodescriptie een natuurlijk onderdeel van de video. Als je in het script rekening houdt met welke informatie er in het audiospoor moet voorkomen, hoef je daarna vaak minder of geen extra audiodescriptie toe te voegen. Bijvoorbeeld:

- Laat de presentator zichzelf en de spreker met naam en functie voorstellen.

> - Laat de sprekers elkaar bij naam noemen, bij het stellen van een vraag.

Reviewer: ```suggestion
- Laat de sprekers elkaar bij naam noemen bij het stellen van een vraag.
```

- Laat sprekers benoemen welke acties ze uitvoeren, als dat op een natuurlijke manier kan.
- Laat de presentator of spreker belangrijke kenmerken van de omgeving benoemen.
- Laat de spreker de tekst in beeld benoemen, bijvoorbeeld in een opname van een webinar waar de spreker gebruikt maakt van tekstslides.

Daarnaast kun je in het script rekening houden met audiodescriptie door te zorgen voor voldoende natuurlijke pauzes:

- Laat de spreker niet direct praten aan het begin van de opname, maar pas na een paar seconden.

> - Stop niet direct met filmen als de spreker dienst tekst heeft afgerond, maar film nog een paar seconden door.

Reviewer: ```suggestion
- Stop niet direct met filmen als de spreker diens tekst heeft afgerond, maar film nog een paar seconden door.
```
Of misschien "als de spreker klaar is met spreken" om zijn/haar/diens te vermijden?

Als je over de oorspronkelijke bestanden beschikt, kun je vrij makkelijk ruimte creëren door de overgangen in de video te verlengen. Bijvoorbeeld door:

- Titelschermen toe te voegen, bijvoorbeeld tussen de hoofdstukken in een video. Ook aan het begin en einde van de video kun je hierdoor makkelijk secondes toevoegen aan je video.
- Overgangseffecten tussen de scènes toe te voegen, bijvoorbeeld door twee stilstaande beelden in elkaar over te laten vloeien, of door het beeld naar zwart te laten vervagen voor de volgende scène begint.
- Extra beeldmateriaal - die je eerder uit de video zou hebben gesneden - nog enkele tellen door te laten lopen.

Wil je muziek toevoegen aan de overgangen? Let er dan wel op dat het volume laag genoeg is om de audiodescriptie duidelijk te kunnen horen.

## Meer lezen over gerelateerde WCAG-succescriteria op NL Design System

> Op de WCAG succescriteria pagina's op deze website lees je meer over veelgemaakte fouten en hoe je kunt testen of jouw website aan de succescriteria voldoet.

Reviewer: Ik zie zowel "WCAG succescriteria" als "WCAG-succescriteria" gebruikt worden op onze site, is er een specifieke die juist is?

## Is audiodescriptie altijd nodig?

> Je voegt audiodescriptie toe aan een video als er informatie zichtbaar is die niet hoorbaar is. Bezoekers die de video niet (goed) kunnen zien missen dan informatie. Dat is niet bij iedere video het geval, bijvoorbeeld:

Reviewer 2: > Bezoekers die de video niet (goed) kunnen zien 

Je kan (niet moet) dit misschien ook nog simpeler zeggen.
> Bezoekers die de video niet zien

Afhankelijk van hoeveel we willen benadrukken dat het vooral nodig is voor mensen met beperkingen, of dat de groep met mensen die er wat aan hebben breder is.
https://en.wikipedia.org/wiki/Curb_cut_effect
